Makefile

```diff
@@ -8,6 +8,19 @@ ARCH_PREFIX := x86_64-elf
 ASM_COMPILER := nasm
 QEMU_SYSTEM := qemu-system-x86_64
 
+ifeq ($(TOOLCHAIN), gcc)
+	X_CC = $(ARCH_PREFIX)-gcc
+	X_LD = $(ARCH_PREFIX)-ld
+else ifeq ($(TOOLCHAIN), clang)
+	X_CC = clang
+	X_LD = ld.lld
+else
+	$(error "Unknown compiler toolchain")
+endif
+
+C_DEBUG_FLAGS := -g \
+				 -DDEBUG=1
```

Comment:
I think that this is caused by my Vim editor, but i think that the indentation is off, can you fix it?